### ISSUE TITLE
Admin-Spawned TG viruses need to be fixed

### ISSUE THREAD

**Issue by Puckaboo2** [HeavensGate/Eternal#305]
Here's the story. Killer released Brainrot to 3 stupid heads who were nothing but drunken, low-RP messes. Az heard from Maline three heads had brainrot, but she took it as a joke because they were quite stupid. However, she tried to fetch the RD, one of the patients who had brain rot, and contracted it, herself.

Maline, who tried to help, also caught the virus. Petrified, they tried radium injections--didn't work. A (stupefied-induced) overdose on Alkysine didn't work (and that shit is slow-metabolizing.) And te virus could not be read in the virus analyzer,

tl;dr, one of the TG viruses is broken. I have a fear other TG viruses are broken, as well. I checked out the code for some of them, and their mechanisms for curing are nowhere near how Bay's mechanisms operate. I believe Paradise has these same viruses and they work fine .I don't wish to break anything, so, if anybody can look in on this, I'd appreciate it.

**Comment by Jacquar** [comment]
We do not use the viruses. The viruses are scary. The very implementation
of viruses in SS13 has always been a bit overly technical for its own
audience, much like genetics, xenoarch, and.. Most systems that don't
involve yiff or violence, in fact.

This is another fix I've no priority for. Content before solving other
people's old problems in systems that don't provide a very stimulating
gameplay experience. Find me more programmers. Hunt, take down, drag home.

**Comment by Puckaboo2** [comment]
The viruses are scawy uwu I'm fine with the common colds, but damn, some of those are freaky scary uwu
